src/fparser/Fortran2003.py

```diff
@@ -4206,7 +4239,14 @@ class Elsewhere_Stmt(StmtBase, WORDClsBase): # R750
     use_names = ['Where_Construct_Name']
     @staticmethod
     def match(string):
-        return WORDClsBase.match('ELSEWHERE', Where_Construct_Name, string)
+        if string[:9].upper() != 'ELSEWHERE' and \
```

Comment:
Now have a single regular expression which is used by both Masked_Elsewhere and Elsewhere.